tui/keys.go

```diff
@@ -44,9 +45,19 @@ func (ui *UI) keyPressed(key *tcell.EventKey) *tcell.EventKey {
 		if err := os.Chdir(ui.currentDirPath); err != nil {
 			panic(err)
 		}
-		shell, ok := os.LookupEnv("SHELL")
-		if !ok {
-			shell = "/bin/bash"
+		var shell string
+		if runtime.GOOS == "windows" {
+			shellbin, ok := os.LookupEnv("COMSPEC")
+			if !ok {
+				shellbin = "C:\\WINDOWS\\System32\\cmd.exe"
```

Comment:
I would prefer to refactor this similar way as the `exec`. Create e.g. function `getShellBin` with two implementations...